### DISCUSSION TITLE
Not able to emit from background thread. Similar to: https://github.com/miguelgrinberg/Flask-SocketIO/issues/434

### DISCUSSION THREAD

**Discussion by Aditya-Dexian** [miguelgrinberg/python-socketio#1358] [Q&A]
Hi,

This problem has been described in https://github.com/miguelgrinberg/Flask-SocketIO/issues/434 as well.

I am a beginner programmer trying to implement a real-time transcription solution using Python Flask, React.js, and websockets.

I am asking for help with my code. 

I seriously appreciate any and all help with this query.

_Note: This code is minimal, not my entire code_

server.py:
```ch in advance.

**Comment by miguelgrinberg** [comment]
Please enable logging in the server (see the troublshooting section if you don't know how) then share logs from your server that include the emits that do not work.

**Comment by Aditya-Dexian** [comment]
I apologize, I think I wrote my question on the wrong repository. I am using flask_socketio, not python_socketio. I will move my question into the correct place.
Thank you for your time and response, Miguel.
